### PR TITLE
Fix links in security context page

### PR DESCRIPTION
**Pull request by tengqm**
The links to AppArmor etc are using redirection records. This is a bad
practice because when translating the page to a different language, the
link will become invalid. So we should use the actual target in the
English source.

This PR also fixes some incorrect syntax directives for code snippets.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-31630--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/security-context/

File: content/en/docs/tasks/configure-pod-container/security-context.md [hunk 1]
```diff
@@ -14,21 +14,29 @@ A security context defines privilege and access control settings for
...
 
-* [AppArmor](/docs/tutorials/clusters/apparmor/): Use program profiles to restrict the capabilities of individual programs.
+* [AppArmor](/docs/tutorials/security/apparmor/):
+  Use program profiles to restrict the capabilities of individual programs.
 
-* [Seccomp](/docs/tutorials/clusters/seccomp/): Filter a process's system calls.
+* [Seccomp](/docs/tutorials/security/seccomp/): Filter a process's system calls.
 
-* AllowPrivilegeEscalation: Controls whether a process can gain more privileges than its parent process. This bool directly controls whether the [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) flag gets set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has `CAP_SYS_ADMIN`.
+* AllowPrivilegeEscalation: Controls whether a process can gain more privileges than
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:32]: nit: Should `AllowPrivilegeEscalation` use code format like `readOnlyRootFilesystem`?

File: content/en/docs/tasks/configure-pod-container/security-context.md [hunk 2]
```diff
@@ -374,7 +386,10 @@ See [capability.h](https://github.com/torvalds/linux/blob/master/include/uapi/li
 for definitions of the capability constants.
 
 {{< note >}}
-Linux capability constants have the form `CAP_XXX`. But when you list capabilities in your Container manifest, you must omit the `CAP_` portion of the constant. For example, to add `CAP_SYS_TIME`, include `SYS_TIME` in your list of capabilities.
+Linux capability constants have the form `CAP_XXX`.
+But when you list capabilities in your Container manifest, you must
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:390]: nit: change Container to lowercase (container)?

File: content/en/docs/tasks/configure-pod-container/security-context.md [hunk 3]
```diff
@@ -127,19 +130,26 @@ ls -l
...
+The output is similar to this:
+
+```none
 uid=1000 gid=3000 groups=2000
 ```
-You will see that gid is 3000 which is same as `runAsGroup` field. If the `runAsGroup` was omitted the gid would
-remain as 0(root) and the process will be able to interact with files that are owned by root(0) group and that have
-the required group permissions for root(0) group.
+
+You will see that gid is 3000 which is same as `runAsGroup` field.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:149]: nit: From the output, you can see that `gid` is 3000 which is the same as the `runAsGroup` field?
or
The `gid` is 3000 which is the same [value] as the `runAsGroup` field.

The rest of the paragraph could use a bit more tidying, but can leave it for another PR.

File: content/en/docs/tasks/configure-pod-container/security-context.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/tasks/configure-pod-container/security-context.md]: I am a little conservative on this. I saw some changes rewriting 'Container' as 'container' even when we are referring to the Container resource (just like a Pod resource), but I don't think those changes are correct.
In this specific case, maybe 'Container' is more appropriate than 'container' because we are talking about the manifest.
Changing this to 'container' is okay though.

Comment (kbhawkey) [reply content/en/docs/tasks/configure-pod-container/security-context.md]: Sure, my comment is more of a note or question.

File: content/en/docs/tasks/configure-pod-container/security-context.md [hunk 4]
```diff
@@ -14,21 +14,29 @@ A security context defines privilege and access control settings for
...
-* [Seccomp](/docs/tutorials/clusters/seccomp/): Filter a process's system calls.
+* [Seccomp](/docs/tutorials/security/seccomp/): Filter a process's system calls.
 
-* AllowPrivilegeEscalation: Controls whether a process can gain more privileges than its parent process. This bool directly controls whether the [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) flag gets set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has `CAP_SYS_ADMIN`.
+* `allowPrivilegeEscalation`: Controls whether a process can gain more privileges than
+  its parent process. This bool directly controls whether the
+  [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+  flag gets set on the container process.
+  `allowPrivilegeEscalation` is true always when the container is:
+  1) run as privileged OR 2) has `CAP_SYS_ADMIN`.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/security-context.md:37]: nit: For some reason, the first item in line 37 looks like an ordered list.
What do you think about pushing these items into an unordered list (in Markdown)?

```
* run as privileged
* has `CAP_SYS_ADMIN`
```

Comment (tengqm) [reply content/en/docs/tasks/configure-pod-container/security-context.md:37]: sure.
